src/unix/osd.c

```diff
@@ -120,6 +120,26 @@ int ofi_wait_cond(pthread_cond_t *cond, pthread_mutex_t *mut, int timeout_ms)
 	return pthread_cond_timedwait(cond, mut, &ts);
 }
 
+int ofi_mmap_anon_pages(void **memptr, size_t size)
+{
+	size = ofi_get_aligned_size(size, ofi_get_page_size());
+	*memptr = mmap(NULL, size, PROT_READ | PROT_WRITE,
+		MAP_PRIVATE | MAP_ANONYMOUS, -1, 0);
+	if (OFI_UNLIKELY(*memptr == MAP_FAILED)) {
+		return -ofi_syserr();
```

Comment:
Shouldn't this be just `errno`?

Reply:
That's what the wrapper returns.  Seeing as this is platform-specific code, sure, I can replace this with a direct use of `errno`.